packages/docusaurus/bin/docusaurus.js

```diff
@@ -238,6 +239,7 @@ function isInternalCommand(command) {
 
 async function run() {
   if (!isInternalCommand(process.argv.slice(2)[0])) {
+    // @ts-expect-error: Hmmm
```

Comment:
?

Reply:
I'm enabling `ts-check`, but there's a type incompatibility. I don't have the bandwidth to investigate why that's the case for now, but I'll probably fix it before merge